.gitignore

```diff
@@ -35,3 +35,5 @@ tags
 /comm_config.yaml
 
 /bin
+
+./botkube
```

Comment:
Actually it is: https://github.com/kubeshop/botkube/blob/b6b7d449278617d40f05d0792b419a7692ad980f/hack/goreleaser.sh#L110

>  but theoretically there are infinite ways of naming the output binary

yes, but you need to agree that naming BotKube binary as `botkube` is probably 80% 😄

Reply:
So that means I'm in 20%, because if I ran `go build` I didn't even provide binary name and was ending up with `main` 😄 

If it is in the Makefile/scripts, then let's keep it 👍